projects/affiliated.md

```diff
@@ -0,0 +1,79 @@
+
+*HSF Affiliated Projects and Software* are community projects or software packages with person-power, and possibly dedicated funding, which connect strongly to the HSF and align with [its remit]({{ site.baseurl }}/organization/goals.html). They benefit from their inclusion in, or association with, the HSF through access to the community for wide visibility and easier promotion through the network. Affiliation makes it clear that the HSF has no control over the endeavours/products, hence no responsibility for evolution or maintenance.
+
+Projects are in general not hosted on the HSF GitHub repository since they are externally managed and maintained, though some are on the [HSF GitHub repository](https://github.com/HSF).
+
+## Engagement and endorsement
+
+Any member of the community is welcome to engage with the HSF in general and the
+[HSF Steering Group (SG)]({{ site.baseurl }}/organization/team.html),
+in particular to discuss the route towards making a software project or package an HSF Affiliated Project or Software. This can be an informal discussion initially.
+
+Formal endorsement is provided by the HSF SG following a short evaluation period in consultation with the relevant HSF Working Group conveners and community experts. For transparency, these evaluations (start, scope and end) are advertised publicly via the HSF Forum, the HSF's main means of communication.
+
+The list of HSF Affiliated Projects and Software is hosted in a dedicated area on the HSF website, curated and maintained by the HSF SG.
+
+The recognition of bijective engagement with the HSF is displayed via *GitHub Badges*.
+Three levels distinguish mainly the level of maturity, community support and  engagement: Bronze, Silver and Gold.
+In broad terms, the attribution of the endorsement level is based on the following guidelines, which are detailed in the dedicated document on Guidelines for HSF Affiliated Projects and Software, and Endorsement Badge Levels.
+* Bronze: new or young endeavour, likely evolving from and within a collaboration or experiment, but with the potential for other communities to use. It should be committed to meeting best practices in software engineering, e.g., documentation and a test suite.
+* Silver: aiming for gold but in an earlier phase towards strong community support (e.g., adoption is still relatively shy, maintenance is not secured at least in the medium term by more than a single person). High standards of software engineering should be met.
+* Gold: endeavour adopted by several collaborations and/or experiments with a strong and long-term community support model.
+
+The HSF envisages a scheme for Annual Project Awards. A concrete proposal will be presented to the HSF Advisory Group once the latter is put in place.
+
+## Best-practice guidelines
+
+They are detailed in the document Guidelines for HSF (Affiliated) Projects and Software.
```

Comment:
why "affiliated" in brackets?

Reply:
Oh, that's a left-over from the times we made a distinction. Thanks for spotting it. Removing the parentheses.